include/envoy/http/header_map.h

```diff
@@ -61,6 +73,10 @@ class LowerCaseString {
   bool operator!=(const LowerCaseString& rhs) const { return string_ != rhs.string_; }
   bool operator<(const LowerCaseString& rhs) const { return string_.compare(rhs.string_) < 0; }
 
+  friend std::ostream& operator<<(std::ostream& os, const LowerCaseString& bar) {
```

Comment:
Probably "bar" is not the right name for this?

Reply:
Renamed.